src/DefinitionResolver.php

```diff
@@ -89,7 +89,7 @@ public function getDocumentationFromNode(Node $node)
         } else {
             $docBlock = $node->getAttribute('docBlock');
             if ($docBlock !== null) {
-                return $docBlock->getSummary();
+                return $docBlock->getSummary() . "\n" . $docBlock->getDescription();
```

Comment:
I think this should be a double line break because it represents a paragraph in markdown and they are separated in the source through a double line break